core/src/main/java/com/chopshop166/chopshoplib/drive/MockSwerveModule.java

```diff
@@ -76,4 +77,14 @@ public Rotation2d getAngle() {
     public SwerveModuleState getState() {
         return this.desiredState;
     }
+
+    @Override
+    public SmartMotorController getDriveMotor() {
+        return null;
```

Comment:
Bad Ben! No `null` allowed! I'd rather you return mock objects.

Reply:
Fine....